tools/expiredChecker.js

```diff
@@ -0,0 +1,30 @@
+function loadJSON(file) {
+  var fs = require('fs');
+  return JSON.parse(fs.readFileSync(file, 'utf8'));
+}
+
+function checkExpiredAll() {
+  let files = ['data/cashback/data.json', 'data/deal/data.json'];
+  files.forEach(function(file) {
+    checkExpiredFile(file);
+  });
+}
+
+function checkExpiredFile(file) {
+  let json = loadJSON(file);
+  let today = new Date();
+  json.forEach(function(deal) {
```

Comment:
Let's change the name to `item`, as the files we are passing into this function are for both `deals` and `cashbacks`. 

This name be consistent with the rest of the source code. For example, see the function `getItemsForUrlOrName` in `src/background.js`

```suggestion
  json.forEach(function(item) {
```